core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java

```diff
@@ -263,6 +265,7 @@ public final void readFrom(StreamInput in) throws IOException {
         public static final String FROM_AS_STRING = "from_as_string";
         public static final String TO = "to";
         public static final String TO_AS_STRING = "to_as_string";
+        public static final ParseField VALUE_TYPE = new ParseField("value_type");
```

Comment:
Ah, so is `InternalAggregation` specifically for the response side? I'm not as intimate with the aggs framework so I thought it was just an internal framework class (request / response agnostic).

So then what do you think about having a `CommonFields` static class to `AggregatorBuilder` for common fields used on the request side?